<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomServiceDialog_q</class>
 <widget class="QWidget" name="CustomServiceDialog_q">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom Service</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame7">
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame14">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Sunken</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="textLabel1">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="obj_name"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="platformLabel">
             <property name="text">
              <string>Platform:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QComboBox" name="platform">
             <property name="toolTip">
              <string>Custom service object has separate code string for each supported firewall platform.</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="codeLabel">
             <property name="text">
              <string>Code:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QLineEdit" name="code">
             <property name="toolTip">
              <string>Custom service object has separate code string for each supported firewall platform.</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="protocolLabel">
             <property name="text">
              <string>Protocol:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QComboBox" name="protocol">
             <property name="toolTip">
              <string>Choose one of the standard protocols from the menu or enter a custom protocol string. Policy compilers can use this information to add correct protocol-specific parameters to the generated configuration.</string>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="afLabel">
             <property name="text">
              <string>Address family:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QRadioButton" name="ipv4">
             <property name="text">
              <string>IPv4</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QRadioButton" name="ipv6">
             <property name="text">
              <string>IPv6</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="CommentKeywords" name="commentKeywords" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommentKeywords</class>
   <extends>QWidget</extends>

   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>obj_name</tabstop>
  <tabstop>platform</tabstop>
  <tabstop>code</tabstop>
  <tabstop>protocol</tabstop>
  <tabstop>ipv4</tabstop>
  <tabstop>ipv6</tabstop>
 </tabstops>
 <resources>
  <include location="MainRes.qrc"/>
 </resources>
</ui>
